unit-test-coverage/pc-rtems/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+######################################################################
+#
+# CMake build recipe for pc-rtems PSP white-box coverage tests
+#
+######################################################################
+
+include_directories(${CFEPSP_SOURCE_DIR}/fsw/pc-rtems/inc)
+include_directories(${PSPCOVERAGE_SOURCE_DIR}/shared/inc)
```

Comment:
Since none of the shared code is tested in this unit (unlike VxWorks version), I think you can remove this include and the other references below.

Reply:
I'm getting undefined ref errors from the testrunner when removing the shared files. Is there some other way around this or do I need to leave them in?
```
/usr/bin/ld: ut-pc-rtems-impl/CMakeFiles/psp-ut-pc-rtems-impl.dir/src/cfe_psp_exception.c.o: in function `CFE_PSP_AttachExceptions':
/home/avi/test/cFS/psp/fsw/pc-rtems/src/cfe_psp_exception.c:69: undefined reference to `CFE_PSP_Exception_Reset'
/usr/bin/ld: ut-pc-rtems-impl/CMakeFiles/psp-ut-pc-rtems-impl.dir/src/cfe_psp_memory.c.o: in function `CFE_PSP_SetupReservedMemoryMap':
/home/avi/test/cFS/psp/fsw/pc-rtems/src/cfe_psp_memory.c:418: undefined reference to `CFE_PSP_MemRangeSet'
/usr/bin/ld: ut-pc-rtems-impl/CMakeFiles/psp-ut-pc-rtems-impl.dir/src/cfe_psp_start.c.o: in function `CFE_PSP_Main':
/home/avi/test/cFS/psp/fsw/pc-rtems/src/cfe_psp_start.c:212: undefined reference to `CFE_PSP_ModuleInit'
collect2: error: ld returned 1 exit status
make[7]: *** [psp/unit-test-coverage/pc-rtems/CMakeFiles/coverage-ut-pc-rtems-testrunner.dir/build.make:210: psp/unit-test-coverage/pc-rtems/coverage-ut-pc-rtems-testrunner] Error 1
make[6]: *** [CMakeFiles/Makefile2:8841: psp/unit-test-coverage/pc-rtems/CMakeFiles/coverage-ut-pc-rtems-testrunner.dir/all] Error 2
make[5]: *** [Makefile:160: all] Error 2
make[4]: *** [CMakeFiles/native_default_cpu1-all.dir/build.make:76: CMakeFiles/native_default_cpu1-all] Error 2
make[3]: *** [CMakeFiles/Makefile2:480: CMakeFiles/native_default_cpu1-all.dir/all] Error 2
make[2]: *** [CMakeFiles/Makefile2:322: CMakeFiles/mission-all.dir/rule] Error 2
make[1]: *** [Makefile:233: mission-all] Error 2
make: *** [Makefile:119: all] Error 2
```

Reply:
Oh, I forgot about those.  Might make sense eventually to add stubs for those shared calls but that can be done later.